mono/metadata/object-internals.h

```diff
@@ -489,7 +490,7 @@ struct _MonoInternalThread {
 
 struct _MonoThread {
 	MonoObject obj;
-	struct _MonoInternalThread *internal_thread;
```

Comment:
Any header switching from its own forward declare to rely on typedef should gain an include.